modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/util/TaxonomyGroupUtil.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.admin.taxonomy.internal.util;
+
+import com.liferay.depot.model.DepotEntry;
+import com.liferay.depot.service.DepotEntryLocalServiceUtil;
+import com.liferay.headless.admin.taxonomy.dto.v1_0.AssetLibrary;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.model.GroupConstants;
+import com.liferay.portal.kernel.service.GroupLocalServiceUtil;
+import com.liferay.portal.kernel.util.ArrayUtil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Adolfo Pérez
+ */
+public class TaxonomyGroupUtil {
+
+	public static long[] getAssetLibraryGroupIds(AssetLibrary[] assetLibraries)
+		throws Exception {
+
+		List<Long> groupIds = new ArrayList<>();
+
+		for (AssetLibrary assetLibrary : assetLibraries) {
+			groupIds.add(_getGroupId(assetLibrary.getId()));
+		}
+
+		return ArrayUtil.toLongArray(groupIds);
+	}
+
+	public static long getCMSGroupId(long companyId) {
+		Group group = GroupLocalServiceUtil.fetchFriendlyURLGroup(
+			companyId, GroupConstants.CMS_FRIENDLY_URL);
```

Comment:
Should we get the CMS group by ERC instead of the friendlyURL since the friendlyURL can be changed?

Reply:
You're right. I should have used the groupKey (the ERC is autogen from it). I'll push the changes to this same PR.